client/src/actions/team.js

```diff
@@ -1,15 +1,70 @@
 /**
  * This file houses our team-related Action Creators.
  */
-import * as api from '../api';
+import * as api from "../api";
 import {
-    LINK_TEAM_TWITTER, 
-    UNLINK_TEAM_TWITTER, 
-    FETCH_TEAM_INFO,     
-    GET_TEAM_MEMBERS_BY_TEAM_ID, 
-    CREATE_TEAM_MEMBER, 
-    UPDATE_TEAM_MEMBER, 
-    DELETE_TEAM_MEMBER } from './types';
+  LINK_TEAM_TWITTER,
+  UNLINK_TEAM_TWITTER,
+  FETCH_TEAM_INFO,
+  GET_TEAM_MEMBERS_BY_TEAM_ID,
+  CREATE_TEAM_MEMBER,
+  UPDATE_TEAM_MEMBER,
+  DELETE_TEAM_MEMBER,
+  ADD_TEAM,
+} from "./types";
+
+/**
+ * Adds a new team to redux store and database. If api request fails the team state's requestFailed attribute will be 'true'
+ * @param teamInfo contains teamName, orgName and emailnot
+ */
+export const addTeamInfo = (teamInfo, result) => async (dispatch) => {
+  try {
+    const data = await api.addTeam(teamInfo);
+    const teamData = {
+      ...teamInfo,
+      teamId: data.data._id,
+      requestFailed: false,
+      error: "",
+    };
+    dispatch({
+      type: ADD_TEAM,
+      payload: teamData,
```

Comment:
I think this dispatch isn't modifying the state, is that intentional?

Reply:
What makes you say that? It seems to update the store since we can see the team name on Dashboard.

Reply:
don't worry, looks like i was misreading